hack/unwanted-dependencies.json

```diff
@@ -216,8 +216,8 @@
       ],
       "golang.org/x/exp": [
         "github.com/antlr4-go/antlr/v4",
-        "github.com/cilium/ebpf",
         "github.com/google/cel-go",
+        "github.com/opencontainers/runc",
```

Comment:
Adding this
```
github.com/opencontainers/runc@v1.2.0 golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
```

Before
```
github.com/opencontainers/runc@v1.1.15 github.com/cilium/ebpf@v0.7.0
github.com/cilium/ebpf@v0.11.0 golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
```

Reply:
aww, I was excited we were dropping ebpf ... can we figure out why runc is using exp and open a PR to help them drop usage?

Reply:
I opened https://github.com/opencontainers/runc/issues/4469.

Reply:
This seems to be a bug of `go mod graph`.

https://github.com/opencontainers/runc/issues/4469#issuecomment-2431238789

```
(base) ➜  runc git:(main) go mod graph                     
github.com/opencontainers/runc golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
github.com/cilium/ebpf@v0.16.0 golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
```

Runc does not use `golang.org/x/exp` directly. As we don't use ` github.com/cilium/ebpf`, `golang.org/x/exp` should be excluded as well.

Reply:
As I mentioned in https://github.com/kubernetes/kubernetes/pull/128276#issue-2606584699, there seems to be a bug.

Reply:
> As we don't use  github.com/cilium/ebpf

It doesn't matter, ebpf is in your dependency tree, so you inherit the dependencies of ebpf.

```
runc]$ git status
HEAD detached at v1.2.0
nothing to commit, working tree clean

runc]$ go mod graph | grep -E 'exp|ebpf'
github.com/opencontainers/runc github.com/cilium/ebpf@v0.16.0
github.com/opencontainers/runc golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
...
github.com/cilium/ebpf@v0.16.0 golang.org/x/exp@v0.0.0-20230224173230-c95f2b4c22f2
```

and pass those on to your dependencies